app/config/default_parameters.yml

```diff
@@ -37,6 +37,12 @@ parameters:
     # Cache DSN, see app/config/services/cache.yml for examples:
     cache_dsn: '%env(CACHE_DSN)%'
 
+    # Content types included in Recommendations from YooChoose account
+    ez_recommendation.default.recommender.included_content_types: [blog, article, place]
+
+    # Location id of the content item for form uploads
+    form_builder.upload_folder.location_id: 250
```

Comment:
This one should be equal to 76, as done here https://github.com/ezsystems/ezplatform-ee-demo/commit/7368dc671d39fe1c945066c54c18422acf8b42ab
(it would be best if @damianz5 can confirm)

Reply:
Then this line should be removed, it should not be two places in the file.

Reply:
fixed